[2021-10-16 13:05:59,468] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_2 scheduled__2021-10-16T12:52:00+00:00 [queued]>
[2021-10-16 13:05:59,503] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: fourth_dag.get_gross_year_2 scheduled__2021-10-16T12:52:00+00:00 [queued]>
[2021-10-16 13:05:59,504] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:05:59,504] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-10-16 13:05:59,504] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-16 13:05:59,543] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): get_gross_year_2> on 2021-10-16 12:52:00+00:00
[2021-10-16 13:05:59,556] {standard_task_runner.py:52} INFO - Started process 1053 to run task
[2021-10-16 13:05:59,584] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fourth_dag', 'get_gross_year_2', 'scheduled__2021-10-16T12:52:00+00:00', '--job-id', '539', '--raw', '--subdir', 'DAGS_FOLDER/fourth_dag.py', '--cfg-path', '/tmp/tmpa2lqg0wp', '--error-file', '/tmp/tmpenoo2b4c']
[2021-10-16 13:05:59,584] {standard_task_runner.py:77} INFO - Job 539: Subtask get_gross_year_2
[2021-10-16 13:05:59,734] {logging_mixin.py:109} INFO - Running <TaskInstance: fourth_dag.get_gross_year_2 scheduled__2021-10-16T12:52:00+00:00 [running]> on host e03fb9e01fd8
[2021-10-16 13:05:59,861] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fourth_dag
AIRFLOW_CTX_TASK_ID=get_gross_year_2
AIRFLOW_CTX_EXECUTION_DATE=2021-10-16T12:52:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-16T12:52:00+00:00
[2021-10-16 13:05:59,898] {python.py:152} INFO - Done. Returned value was: ['$29.37 ', '2007\n']
[2021-10-16 13:06:00,085] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=fourth_dag, task_id=get_gross_year_2, execution_date=20211016T125200, start_date=20211016T130559, end_date=20211016T130600
[2021-10-16 13:06:00,186] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-10-16 13:06:00,285] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
